joomla/joomla-cms
# [com_banners] Add search tooltip to all views

#### Summary of Changes

Simple PR to improve the search tooltip and element label (accessibility) in com_banners.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Components -> Banners -> Banners , check the tooltip in the search field.
3. Go to Components -> Banners -> Clients, check the tooltip in the search field.
4. Go to Components -> Banners -> Tracks, check the tooltip in the search field.
